docker-compose.yml

```diff
@@ -45,3 +45,5 @@ services:
       - DATASOURCE_DB_SCHEMA=public
       - DATASOURCE_DB_PORT=5432
       - DATASOURCE_DB_HOST=db
+      - USERNAME=username
+      - PASSWORD=password
```

Comment:
Can we name these more clearly, so they don't get confused with e.g. the db credentials, or any others?

probably `TASK_API_USERNAME` etc.?